modules/SofaMiscFem/FastTetrahedralCorotationalForceField.inl

```diff
@@ -155,6 +155,11 @@ template <class DataTypes> FastTetrahedralCorotationalForceField<DataTypes>::Fas
     , f_youngModulus(initData(&f_youngModulus,(Real)1000.,"youngModulus","Young modulus in Hooke's law"))
     , lambda(0)
     , mu(0)
+    , f_drawing(initData(&f_drawing, true, "drawing", " draw the forcefield if true"))
+    , drawColor1(initData(&drawColor1, defaulttype::Vec4f(0.0f, 0.0f, 1.0f, 1.0f), "drawColor1", " draw color for faces 1"))
```

Comment:
Why not a vector of color instead of 4 data ?

Reply:
same answer as above.